python/res/enkf/enkf_simulation_runner.py

```diff
@@ -41,3 +82,39 @@ def runWorkflows(self , runtime):
         """:type res.enkf.enum.HookRuntimeEnum"""
         hook_manager = self._enkf_main().getHookManager()
         hook_manager.runWorkflows(runtime  , self._enkf_main())
+
+
+    def add_job(self, run_arg, res_config, job_queue):
+        job_name = run_arg.job_name
+        run_path = run_arg.runpath
+        job_script = res_config.queue_config.job_script
+        num_cpu = res_config.queue_config.num_cpu
+        if num_cpu == 0:
+            num_cpu = res_config.ecl_config.num_cpu
+        
+        job = JobQueueNode(job_script=job_script, job_name=job_name, run_path=run_path, num_cpu=num_cpu,
+                            status_file=job_queue.status_file, ok_file=job_queue.ok_file, exit_file=job_queue.exit_file,
+                            done_callback_function=EnKFState.forward_model_ok_callback,
+                            exit_callback_function=EnKFState.forward_model_exit_callback,
+                            callback_arguments=[run_arg, res_config])
+        
+        if job is None:
+            return
+        job_queue.add_job(job)
```

Comment:
See https://github.com/ManInFez/libres/pull/1/files#diff-bf3b2576085be860275517eae229f742R103

I think we should keep setting the `queue_index`, which will make `detailed_progress` work as it did.